<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TUIcomotiveClass</class>
 <widget class="QMainWindow" name="TUIcomotiveClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>298</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>TUIcomotive</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>232</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Banga list</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListWidget" name="bangaList"/>
     </item>
     <item row="1" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>skADDje</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>sletXIT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="testButton">
         <property name="text">
          <string>UI Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="menuButton">
         <property name="text">
          <string>Menu Button</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="styleButton">
         <property name="text">
          <string>Style Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="canButton">
         <property name="text">
          <string>CAN test</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>128</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>&lt;No item selected&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>366</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TUIcomotive.qrc"/>
 </resources>
 <connections/>
</ui>
